doc/dev/osd_internals/erasure_coding/ecbackend.rst

```diff
@@ -149,23 +149,23 @@ written. The RADOS operation is acknowledged.
 OSD Object Write and Consistency
 --------------------------------
 
-Regardless of the algorithm chosen above, writing of the data is a two
+Regardless of the algorithm chosen above, writing of the data is a two-
 phase process: commit and rollforward. The primary sends the log
 entries with the operation described (see
 osd_types.h:TransactionInfo::(LocalRollForward|LocalRollBack).  
 In all cases, the "commit" is performed in place, possibly leaving some
 information required for a rollback in a write-aside object.  The
-rollforward phase occurs once all acting set replicas have committed
-the commit (sorry, overloaded term) and removes the rollback information.
+rollforward phase occurs once all acting set replicas have finalized(committed)
```

Comment:
i'd prefer using "commit" instead of "finalize" even we are using this word as both verb and a noun in the same sentence. i think we are used to put it on daily communication.

Reply:
Changed